src/libraries/System.Security.Cryptography/src/System.Security.Cryptography.csproj

```diff
@@ -139,6 +139,7 @@
       <Link>Common\System\Security\Cryptography\Asn1\GeneralNameAsn.xml.cs</Link>
       <DependentUpon>Common\System\Security\Cryptography\Asn1\GeneralNameAsn.xml</DependentUpon>
     </Compile>
+    <!-- TODO is this intentionally linked under System\Security\Cryptography instead of Common/System\Security\Cryptography? -->
```

Comment:
No, all of the MLKem things should be linked under Common. You don't have to fix this here - I can do it as a follow up.